.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+    ["@babel/preset-env", { "targets": "> 1%" }],
+    "@babel/preset-react"
+  ]
+}
```

Comment:
babel is needed to properly compile es6 (and TS in the future) into the jest environment